default.style

```diff
@@ -1,232 +1,279 @@
-node,way   generator:source    text  linear
-node,way   harbour      text         polygon
-node,way   highway      text         linear
-node,way   historic     text         polygon
-node,way   horse        text         linear
-node,way   intermittent text         linear
-node,way   junction     text         linear
-node,way   landuse      text         polygon
-node,way   layer        text         linear
-node,way   leisure      text         polygon
-node,way   lock         text         linear
-node,way   man_made     text         polygon
-node,way   military     text         polygon
-node,way   motorcar     text         linear
-node,way   name         text         linear
-node,way   natural      text         polygon  # natural=coastline tags are discarded by a hard coded rule in osm2pgsql
-node,way   office       text         polygon
-node,way   oneway       text         linear
-node,way   operator     text         linear
-node,way   place        text         polygon
-node,way   population   text         linear
-node,way   power        text         polygon
-node,way   power_source text         linear
-node,way   public_transport text     polygon
-node,way   railway      text         linear
-node,way   ref          text         linear
-node,way   religion     text         linear
-node,way   route        text         linear
-node,way   service      text         linear
-node,way   shop         text         polygon
-node,way   sport        text         polygon
-node,way   surface      text         linear
-node,way   toll         text         linear
-node,way   tourism      text         polygon
-node,way   tower:type   text         linear
-way        tracktype    text         linear
-node,way   tunnel       text         linear
-node,way   water        text         polygon
-node,way   waterway     text         polygon
-node,way   wetland      text         polygon
-node,way   width        text         linear
-node,way   wood         text         linear
-node,way   z_order      int4         linear # This is calculated during import
-way        way_area     real         linear # This is calculated during import
+# If importing with both --hstore and --extra-attributes, the meta-data will
+# end up in the tags hstore column, regardless of the style file.
+#
+# Wiki pages
+# ==========
+#
+# In general, the OpenStreetMap wiki contains a page named for each tag or
+# tag set.  For example:
+#
+#   access          http://wiki.openstreetmap.org/wiki/Key:access
+#   addr:housename  http://wiki.openstreetmap.org/wiki/Key:addr
+#   note:*          http://wiki.openstreetmap.org/wiki/Key:note
+#   tower:type      http://wiki.openstreetmap.org/wiki/Key:tower:type
+#
+# A few tags (e.g., abandoned, disused) are treated as lifecycle prefixes.
+# Their wiki pages contain a terminating colon (:).  For example:
+#
+#   abandoned       http://wiki.openstreetmap.org/wiki/Key:abandoned:
+#   disused         http://wiki.openstreetmap.org/wiki/Key:disused:
```

Comment:
I'm not convinced that the style file is the right place to explain the OSM tagging system. A one liner like "For an see the OSM wiki at https://wiki.openstreetmap.org." should be sufficient and would be less distracting.